src/pg/sql/getArgSql.js

```diff
@@ -38,9 +38,11 @@ export default function getArgSql(
 
   let key;
   const where = [];
-  if (!isEmpty(filter)) {
-    where.push(getFilterSql(filter, options));
-    key = sql`${JSON.stringify(filter)}::jsonb`;
+  if (!isEmpty(filter) || $group === true) {
+    if (!isEmpty(filter)) {
+      where.push(getFilterSql(filter, options));
+    }
+    key = sql`${JSON.stringify({ ...filter, $group })}::jsonb`;
   }
```

Comment:
We can remove the nested if by:
```suggestion
  if (!isEmpty(filter)) where.push(getFilterSql(filter, options));
  if (!isEmpty(filter) || $group === true) {
    key = sql`${JSON.stringify({ ...filter, $group })}::jsonb`;
  }
```

Reply:
Good find!